Makefile

```diff
@@ -51,6 +51,6 @@ scan:
 
 ## test - Test the project
 test:
-	mvn --batch-mode install -Dgpg.skip=true -Dcheckstyle.skip=true -Dcheckstyle.skip=true -Ddependency-check.skip=true -Djavadoc.skip=true
+	mvn surefire:test
```

Comment:
## Author's Note

Same as the change we made on EasyPost Java, the `make test` step now truly just runs the tests, rather than recompiling the source code first. This is important since we explicitly split the compile and run test steps on CI.